src/components/CurrentVideoInfo.jsx

```diff
@@ -1,51 +1,24 @@
 import React from 'react';
+import { Textfit } from 'react-textfit';
 
 function CurrentVideoInfo(props) {
-  const titleOpts = {
-    flex: 1
+  var titleOpts = {
+    flex: 1,
+    height: "110px",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center" 
   }
   const anchorOpts = {
     paddingLeft: 3, 
     paddingRight: 10,
     flex: 0
   }
 
-  function fitText(outputSelector){
```

Comment:
Did I write this? I legit have 0 memory of this code lol

Reply:
NAh I added and removed it within my own set of commits

Reply:
Oh I see oops I was looking at a single commit instead of the newest version of the PR